### PR TITLE
Protocol check fix in _tryGetOrigin

### PR DESCRIPTION
**Pull request by IhabAlkabary**

### CODE REVIEW

**Comment by IhabAlkabary** [comment]
@omsmith @AKobets

**Comment by IhabAlkabary** [comment]
Looking at the testcase `should resolve protocol-relative origin` is this expected?
I thought either there is a protocol or there isn't, but do we expect // at the beginning?

**Comment by omsmith** [comment]
Yes this is expected. `//d2l.com means `https://d2l.com` if I'm on an HTTPS site already, and similar for HTTP. It's called protocol-relative.

What prompted you to make this change?

File: src/host.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ Host._createIFrame = function createIFrame(src, frameId, height, allowFullScreen
 };
 
 Host._tryGetOrigin = function tryGetOrigin(url) {
-	if (url && url.indexOf('//') === 0) {
+	if (url && url.indexOf('//') === -1) {
```
Comment (AKobets) [inline src/host.js]: are there unit tests?

Comment (AKobets) [inline src/host.js]: Should we add another if statement for `url && url.indexOf('//') === -1` case?

Comment (IhabAlkabary) [reply src/host.js]: yeah, and failing with this change. I like the comment you put regarding allow both cases, if the proceeding // is a legitimate case

Comment (IhabAlkabary) [reply src/host.js]: @omsmith We're using ifrau in this context:
https://github.com/Brightspace/what-the-faq/blob/master/Projects/ManagerRole/Local-dev-setup.md#setup-local-fra
and if we omit the protocol altogether (also no //) things just don't work, so it seems logical to add an extra case were when the user does not add a protocol altogether, agree?

Comment (omsmith) [reply src/host.js]: Sorry, don't understand why that should change anything.

What are you trying to pass in as a `src` to host?

**Pull request closed without merging**